src/assets/icons/default-transaction-icon-large/default-transaction-icon-large.tsx

```diff
@@ -2,17 +2,18 @@ import { Component, h } from '@stencil/core';
 
 @Component({
   tag: 'mvx-default-transaction-icon-large',
+  styleUrl: 'default-transaction-icon-large.scss',
   shadow: true,
 })
 export class DefaultTransactionIconLarge {
   render() {
     return (
       <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 41" fill="none">
         <g clip-path="url(#clip0_8136_11348)">
-          <rect y="0.319824" width="40" height="40" rx="20" fill="#787878" fill-opacity="0.2" />
+          <rect y="0.319824" width="40" height="40" rx="20" class={'default-transaction-icon-large-container'} />
```

Comment:
This is not the correct approach. We shouldn't have classes on specific `SVG` components like `rect` or `path`, instead we should pass it to the most upper level of the HTML, which in this case is the `svg` tag itself. Look up other icons that have a custom class and an inherited class from the `@Prop()` and do it the same here. And then in CSS we simply target by element selector these parts of the `SVG`, using selectors like `path {}` and so on.

Reply:
Awesome. 👍